app/helpers/patients_helper.rb

```diff
@@ -73,4 +73,11 @@ def dcaf_pledge_limit_help_text
 
     "Pledge Limit Guidelines:<br />1st trimester (7-12 weeks): $#{first_tri}<br />2nd trimester (12-24 weeks): $#{second_tri}<br />Later care (25+ weeks): $#{later_care}"
   end
+
+  def duration_since_last_call
```

Comment:
Since we're going to ask for this to return days since last call as an int, should we rename it a little to reflect that? Maybe we straight up rename it `days_since_last_call`? (Note that we might have to rename the days variable within it, if that's the case.)